recipe/meta.yaml

```diff
@@ -13,7 +13,9 @@ source:
     - win32.patch  # [win32]
 
 build:
-  number: 0
+  number: 1
+  run_exports:
+    - libsvm >=334,<400
```

Comment:
you should also remove `flatversion` from the meta.yaml, it is the same as `version` since for the last 6 years.
https://github.com/conda-forge/libsvm-feedstock/pull/6